src/System.Runtime.Extensions/ref/System.Runtime.Extensions.cs

```diff
@@ -893,6 +893,9 @@ public abstract partial class StringComparer : System.Collections.Generic.ICompa
         public static System.StringComparer InvariantCultureIgnoreCase { get { throw null; } }
         public static System.StringComparer Ordinal { get { throw null; } }
         public static System.StringComparer OrdinalIgnoreCase { get { throw null; } }
+#if netcoreapp
+        public static System.StringComparer FromComparison(System.StringComparison comparisonType) { throw null; }
+#endif
```

Comment:
You did the right thing by naming the file..netcoreapp.. and conditionally compiling the file. We don't use versions anymore.